lib/android_apk.rb

```diff
@@ -248,7 +248,7 @@ def icon_file(dpi = nil, want_png = false)
   end
 
   def available_png_icon
-    png_path = DPI_TO_NAME_MAP.keys.sort { |i| i }
+    png_path = DPI_TO_NAME_MAP.keys.sort { |l, r| r - l }
```

Comment:
this was a bug 😱 but no effect happened because sorted arrays are not affected by this bug fortunately.